### PR TITLE
LPS-106199 Single experience mode

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/index.js [hunk 1]
```diff
@@ -46,7 +46,10 @@ export default class Experience {
 		this.toolbarId = app.config.toolbarId;
 		this.toolbarPluginId = toolbarPlugin.toolbarPluginId;
 
-		if (app.store.availableSegmentsExperiences !== null) {
+		if (
+			app.store.availableSegmentsExperiences !== null ||
+			app.config.singleSegmentsExperienceMode === true
+		) {
```
Comment (julien) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/index.js]: couldn't this be 

```javascript
if (
  app.store.availableSegmentsExperiences  ||
  app.config.singleSegmentsExperienceMode
) {
```

**Comment by dgarciasarai** [comment]
Pull request submitted here https://github.com/brianchandotcom/liferay-portal/pull/83841.
Thanks @p2kmgcl @julien @andresfulla :)

**Pull request closed without merging**